PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -5744,6 +5798,10 @@ function Show-SddcDiagnosticReport
     # Extract ZIP if neccesary
     $Path = Check-ExtractZip $Path
 
+    # Repror Output file

```

Comment:
No. We don't drop an unsolicited transcript into the current directory. If you'd like to add an option to take a path to tee the output to that would be reasonable. Perhaps if the path is null then autogenerate a path like this ... but it has to be solicited.

Reply:
Looks like, we already transcript in 0_CloudHealthSummary.log, while collecting logs.
Should I just remove the transcript here, and update the "Show-SddcDiagnosticReport -Report Summary -ReportLevel Full $Path" at line 2738 to "Show-SddcDiagnosticReport -Report All -ReportLevel Full $Path"?